src/main/scala/com/wanari/tutelar/core/ConfigService.scala

```diff
@@ -36,4 +37,5 @@ trait ConfigService {
   implicit def ldapConfig: LdapConfig
   implicit def passwordSettings: PasswordSettings
   implicit def escherConfig: EscherConfig
+  implicit def jaegerConfig: Configuration
```

Comment:
I love to hate when a lib calls its inner things so common like `Configuration`.
Can we have a typealias to that somewhere like `type JaegerConfig = io.jaegertracing.Configuration` and use `JaegerConfig` in our codebase everywhere?

Reply:
I had exactly the same thoughts when I saw the ConfigService traitl. I was actually thinking about wrapping it in the class. Would you consider that an option?

Reply:
I think the simple typealias is enough (but the wrapper is fine too).
And the typealias or the wrapper should be in the `TracerService` companion object.

Reply:
updated my PR